communication/data_logging.hpp

```diff
@@ -280,15 +299,58 @@ class Data_logging
     State& state_;                              ///< The pointer to the state structure
 };
 
+
+/**
+ * \brief   Log data in files
+ *
+ * \tparam  N   Maximum number of variables to log
+ */
+template<uint32_t N>
+class Data_logging_tpl: public Data_logging
```

Comment:
Changed the suffix of all class templates to _T in https://github.com/lis-epfl/MAVRIC_Library/pull/326/commits/eba4b84c199b32b2f0cf368c85b62f0851c6b0db